src/DropdownMenu/DropdownButton.tsx

```diff
@@ -7,7 +7,7 @@ export type DropdownButtonProps = ButtonProps
 
 export const DropdownButton = React.forwardRef<HTMLElement, React.PropsWithChildren<DropdownButtonProps>>(
   ({children, ...props}: React.PropsWithChildren<DropdownButtonProps>, ref): JSX.Element => (
-    <Button ref={ref} {...props}>
+    <Button ref={ref} type="button" {...props}>
```

Comment:
If this is a standard html button, we shouldn't have to define the type should we?

Reply:
I think an HTML button defaults to `type="submit"` which is for form submission, but for javascript-y behavior, we'd want to specify `type="button"`.